packages/plugin-ext/src/hosted/browser/worker/worker-env-ext.ts

```diff
@@ -34,4 +34,8 @@ export class WorkerEnvExtImpl extends EnvExtImpl {
         throw new Error('There is no app root in worker context');
     }
 
+    get isNewAppInstall(): boolean {
+        throw new Error('Cannot determine the installation date in worker context');
```

Comment:
Suggestion: Since [vscode](https://github.com/microsoft/vscode/blob/edc585edf6ec34d121196a3eda229c1c0fed676e/src/vs/workbench/api/common/extHost.api.impl.ts#L325-L328) never throws here, we should probably just return `false` to keep better compatibility with vscode.